packages/wrangler/src/deployment-bundle/bundle.ts

```diff
@@ -44,7 +48,27 @@ export const COMMON_ESBUILD_OPTIONS = {
 } as const;
 
 // build conditions used by esbuild, and when resolving custom `import` calls
-export const BUILD_CONDITIONS = ["workerd", "worker", "browser"];
+export function getBuildConditions() {
+	const envVar = getBuildConditionsFromEnv();
+	if (typeof envVar === "string") {
+		return envVar.split(",");
+	} else {
+		return ["workerd", "worker", "browser"];
+	}
+}
+
+function getBuildPlatform(): esbuild.Platform {
+	const platform = getBuildPlatformFromEnv();
+	if (
+		platform !== undefined &&
+		!["browser", "node", "neutral"].includes(platform)
+	) {
+		throw new UserError(
+			"Invalid esbuild platform configuration defined in the WRANGLER_BUILD_PLATFORM environment variable."
```

Comment:
should we tell users what the supported values are here? Otherwise there's no clear path to action

Reply:
Sure. I can add that.

Reply:
Added 76043b0b1